### PR TITLE
Monitoring mockup backend functions

### PR DESCRIPTION
**Pull request by freedgit**
Adding monitoring mockup API calls.

It paves the way to integrate real API calls (to OpenStack, to IPMI data gathering etc) by filling in the blanks as functionality emerges in other parts of the backend/API.

### Checklist

- [ X ] Related tests were updated
- [ ] Related documentation was updated
- [ X ] OpenAPI file was updated
- [ X ] Run `tox`, no tests failed

### CODE REVIEW

File: src/openapi/monitor.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      memory_available:
+        type: integer
+      cpus_used:
+        type: integer
+      memory_used:
+        type: integer
+
+
+endpoints:
+  bm_hosts_to_monitor:
```
Comment (guioliveirabh) [inline src/openapi/monitor.yml:51]: I think you need to add the operation ([RHUB - Code Etiquette Agreement](https://docs.google.com/document/d/1JMWj7Vh6bJeo16kXzCzZvHvHrF34ZgLpF0vwnKSf7cE/edit?usp=sharing)).

Like:
```suggestion
  bm_hosts_to_monitor_list:
```

File: src/rhub/api/monitor.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import os
```
Comment (guioliveirabh) [inline src/rhub/api/monitor.py:1]: Please, move this and the monitoring_*.yml to a directory, to keep them together.

File: src/rhub/api/utils.py [hunk 3]
```diff
@@ -64,3 +65,7 @@ def condition_eval(expr, params):
     elif expr[0] == 'param_in':
         return expr[1] in params and expr[2] in params[expr[1]]
     raise ValueError(f'Unknown operation {expr[0]!r}')
+
+
+def choose_from(number):
```
Comment (fpob) [inline src/rhub/api/utils.py:70]: [`random.randrange`](https://docs.python.org/3/library/random.html?highlight=random#random.randrange)

Comment (freedgit) [reply src/rhub/api/utils.py]: @fpob Fixed in the last commit.

File: src/rhub/api/monitor.py (hunk 2 above)
Comment (freedgit) [reply src/rhub/api/monitor.py]: Thanks, looking better now. Fixed in the last commit.

File: src/openapi/monitor.yml (hunk 1 above)
Comment (freedgit) [reply src/openapi/monitor.yml]: Right. Changed others to _get then, it's indeed more clear.

**Review by guioliveirabh — DISMISSED** [review]

**Review by fpob — DISMISSED** [review]

**Pull request merged**